app/main.py

```diff
@@ -1,6 +1,18 @@
-from typing import Callable
+from typing import Callable, Any
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    inner_cache = {}
+
+    def inner(*args) -> Any:
```

Comment:
Decorator should accept both positional arguments (*args) and keyword arguments (**kwargs).